alooma/alooma.py

```
@@ -1543,6 +1543,18 @@ def get_queries(self):
         url = self.rest_url + endpoints.CONSOLIDATION_V2
         return self.__send_request(requests.get, url).json()
 
+    def get_query_by_event_type(self, event_type):
+    	""" Return scheduled query by event type """
+        params = {"event_type": event_type}
+        url = self.rest_url + endpoints.CONSOLIDATION_V2
+        res = self.__send_request(requests.get, url, params=params)
+
+        cons = res.json()
+        if not cons:
+            raise Exception("Consolidation DNE: %s" % event_type)
+
+        return cons[0]
```

I'd also throw an exception if there is more than 1 that returns